.github/workflows/ci.yml

```
@@ -15,33 +15,24 @@ jobs:
       - name: Generate cache key
         id: cache-key
         run: |
-          echo "key=${{ runner.os }}-turbo-${{ github.sha }}-${{ hashFiles('**/pnpm-lock.yaml', '**/turbo.json', 'turbo.json') }}" >> $GITHUB_OUTPUT
-      
-      - name: Get pnpm store directory
-        id: pnpm-cache
-        shell: bash
-        run: |
-          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
+          echo "key=${{ runner.os }}-turbo-${{ github.sha }}-${{ hashFiles('**/bun.lock', '**/turbo.json', 'turbo.json') }}" >> $GITHUB_OUTPUT
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)